pkg/testcase/cluster.go

```diff
@@ -37,8 +37,8 @@ func TestBuildCluster(cluster *shared.Cluster) {
 	err = shared.PrintBase64Encoded(shared.KubeConfigFile)
 	Expect(err).NotTo(HaveOccurred(), err)
 
-	if cluster.GeneralConfig.BastionIP != "" {
-		shared.LogLevel("info", "Bastion Node IP: %v", cluster.GeneralConfig.BastionIP)
+	if cluster.BastionConfig.PublicIPv4Addr != "" {
+		shared.LogLevel("info", "Bastion Node IP: %v", cluster.BastionConfig.PublicIPv4Addr)
```

Comment:
Do we need this as we have it here https://github.com/rancher/distros-test-framework/pull/130/files#diff-0ebfcc2dc237baa91d60b8e40149c3fc961fddcbc345981fadf0bd26b1948c80R20

Reply:
Yes we need this as they are residing in separate functions